hw_5/point_hw5.py

```diff
@@ -0,0 +1,29 @@
+class Circle:
+    def __init__(self, x=0, y=0, radius=10):
+        self.x = x
+        self.y = y
+        self.radius = radius
+        print(self.x, self.y, self.radius)
+
+    def in_circle(self, f):
+        print(f.x,  f.y)
+        return (self.x - f.get_x()) ** 2 + (self.y - f.get_y()) ** 2 < self.radius ** 2
```

Comment:
you can use just atr of f class, like f.x or f.y

Reply:
fix